### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by kevin-10**
Changed "armorer" to "armourer" per EA.
Plague Beast lord now uses poison spit ability

Enhancements:
consolidated dragon breath/area poison data in the respective SpecialAbility/AreaEffect Class

### CODE REVIEW

File: Scripts/Services/Pet Training/AreaEffects.cs [hunk 1]
```diff
@@ -411,17 +403,19 @@ public PoisonBreath()
 
         public override void DoEffect(BaseCreature creature, Mobile m)
         {
-            m.ApplyPoison(creature, creature.HitAreaPoison);
+            var def = PoisonBreathDefinition.GetDefinition(creature);
+
+            m.ApplyPoison(creature, def.Poison);
```
Comment (Argalep) [inline Scripts/Services/Pet Training/AreaEffects.cs]: The level of poison applied is proportional to the poisoning skill of the creature.

exm: 

```
int level = 1;
            double total = creature.Skills[SkillName.Poisoning].Value;

            if (total >= 100)
                level = 5;
            else if (total > 60)
                level = 4;
            else if (total > 40)
                level = 3;
            else if (total > 20)
                level = 2;
            else if (total > 0)
                level = 1;

m.ApplyPoison(creature, Poison.GetPoison(level));  
```

Comment (kevin-10) [reply Scripts/Services/Pet Training/AreaEffects.cs]: Nothing was changed, the poison level/damage is just now consolidated in one script. This is area poison, not a direct poison attack.

Comment (kevin-10) [reply Scripts/Services/Pet Training/AreaEffects.cs]: According to UOGuide: http://www.uoguide.com/Poison_Breath

Comment (kevin-10) [reply Scripts/Services/Pet Training/AreaEffects.cs]: https://uo.com/wiki/ultima-online-wiki/skills/animal-taming/animal-training-abilities/

Comment (Argalep) [reply Scripts/Services/Pet Training/AreaEffects.cs]: I tested it at the OSl. I taught poison breath to the creature of Triton. He had 20 skills. Level 1 poison affected the player he fought.

Comment (kevin-10) [reply Scripts/Services/Pet Training/AreaEffects.cs]: Ok! nevermind what I say lol.

Comment (Argalep) [reply Scripts/Services/Pet Training/AreaEffects.cs]: tasanar's account is inactive. I was going to share a screenshot.

Comment (Argalep) [reply Scripts/Services/Pet Training/AreaEffects.cs]: Also, Poison breath cannot be taught unless the creatures are taught poisoning

**Pull request merged**